projects/pgai/tests/vectorizer/test_vectorizer_cli.py

```diff
@@ -0,0 +1,420 @@
+import logging
+from collections.abc import Generator
+from typing import Any
+
+import openai
+import psycopg
+import pytest
+from click.testing import CliRunner
+from psycopg import sql
+from psycopg.rows import dict_row
+from testcontainers.postgres import PostgresContainer  # type: ignore
+
+from pgai.cli import vectorizer_worker
+from tests.vectorizer import expected
+
+
+@pytest.fixture(scope="session")
+def cli_postgres_container():
+    """Provides a clean Postgres container for CLI testing"""
+    with PostgresContainer("timescale/timescaledb-ha:pg16", driver=None) as postgres:
+        yield postgres
+
+
+@pytest.fixture
+def cli_db(
+    cli_postgres_container: PostgresContainer,
+) -> Generator[dict[str, Any], None, None]:
+    """Creates a test database with pgai installed"""
+    role = "tsdbquerier"
+    password = "my-password"
+    db_host = cli_postgres_container._docker.host()  # type: ignore
```

Comment:
Do you know if this is a docker for mac workaround, and if it's necessary? I'm not sure it is.

Reply:
We need that for running the tests from the the github action runners containers.